README.md

```diff
@@ -23,16 +24,16 @@ The hardest part of running an all volunteer open source project organization is
 ## How to contribute
 
 1. Join our organization by going through [Hack for LA onboarding][HfLA onboarding]. It's free to join!
-2. Read the [onboarding section of our WIKI](https://github.com/hackforla/peopledepot/wiki/Developer-Onboarding).
-3. Read our [Contributing Guide][contributing] and follow the steps.
+2. Read the [onboarding section of our Wiki](https://github.com/hackforla/peopledepot/wiki/Developer-Onboarding).
+3. Read our [Contributing Guidelines][contributing] and follow the instructions there.
 
 ## Contact info
 
-Contact us in the people-depot channel on Slack.
+Contact us in the `#people-depot` channel on Slack.
 
 ## Licensing
 
-Include details about the project's open source status.
+Open source licenses enable others to freely use, change, and distribute the project in your repository. For more information on repository licenses, see "[Licensing a repository][licensing-a-repo]." This repository uses the [GNU General Public License (v2.0)][licensing].
```

Comment:
we might even consider just using the last part to keep it simple.

Reply:
Cool. A or B?

A.

> For more information on repository licenses, see "[Licensing a repository][licensing-a-repo]." This repository uses the [GNU General Public License (v2.0)][licensing].

B.

> This repository uses the [GNU General Public License (v2.0)][licensing].

Reply:
I think B is better since it's shorter